docs/peopleops/recruiting/index.md

```diff
@@ -0,0 +1,4 @@
+# Recruiting
+
+- [Pitch](pitch.md)
+- [Recruitment Ad](recruitment-ad.md)
```

Comment:
**thought:** These two docs are very similar. Should we drop one of them? 🤔

Reply:
Yes, probably should merge them into one marketing file with both ad and pitch

Reply:
Coolio. I will merge them in a follow-up PR.